code/modules/bitrunning/virtual_domain/virtual_domain.dm

```diff
@@ -74,12 +74,33 @@
 	var/list/custom_spawns = list()
 	/// Set TRUE if you want reusable custom spawners
 	var/keep_custom_spawns = FALSE
-
+	/// At least this many candidates or it will cancel the loading!
+	var/mission_min_candidates = 0
+	/// Maximum amount possible of above.
+	var/mission_max_candidates = 0
```

Comment:
You should say what "candidates" are
`
/// The domain must have this many ghost candidates willing to play hostile entities, or else it will not load.
`